staging/src/k8s.io/code-generator/examples/apiserver/apis/core/v1/doc.go

```diff
@@ -19,5 +19,6 @@ limitations under the License.
 // +k8s:defaulter-gen=TypeMeta
 // +k8s:conversion-gen=k8s.io/code-generator/examples/apiserver/apis/core
 // +groupName=
+// +k8s:openapi-model-package=io.k8s.code-generator.examples.apiserver.apis.core.v1
```

Comment:
question, if someone add a new api group in the future , will the CI inform that it require to set this tag? or is it possible we end merging code without this generator?